README.md

```diff
@@ -41,20 +27,74 @@ blank, the script will remove that JSON element from the record.
 }
 ```
 
-## Operational Note ##
+## Operational note ##
 
 This script will attempt to extract the RVA ID from the `RVA ID` field in a record
 after field mapping has taken place. It expects the ID found to end in the format
 `DDDD`, but allows an increment such that `0123.4` is valid. However, the matched
 ID is reduced to the four leading digits in this case.
 
-## Docker Note ##
+## Building the base Lambda image ##
+
+The base Lambda image can be built with the following command:
+
+```console
+docker compose build
+```
+
+This base image is used both to build a deployment package and to run the
+Lambda locally.
+
+## Building a deployment package ##
+
+You can build a deployment zip file to use when creating a new AWS Lambda
+function with the following command:
+
+```console
+docker compose up build_deployment_package
+```
+
+This will output the deployment zip file in the root directory.
+
+## Running the Lambda locally ##
 
-Please note that the corresponding Docker image _must_ be rebuilt
-locally if the script `build.sh` changes.  Given that rebuilding the Docker
-image is very fast (due to Docker's caching) if the script has not changed, it
-is a very good idea to _always_ run the `docker-compose build` step when
-using this tool.
+The configuration in this repository allows you run the Lambda locally for
+testing as long as you do not need explicit permissions for other AWS
```

Comment:
I think this word is slightly better here, but please disregard if you disagree.
```suggestion
testing as long as you do not require explicit permissions for other AWS
```

Reply:
This is pulled in from the skeleton so I will punt the change upstream.